src/pages/TrueSightV2/components/KyberAIWidget.tsx

```diff
@@ -0,0 +1,502 @@
+import SimpleTooltip from './SimpleTooltip'
+import { WidgetMobileTable, WidgetTable } from './table/index'
+
+const CloseButton = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  border-radius: 100%;
+  height: 16px;
+  width: 16px;
+  background-color: ${({ theme }) => theme.background};
+  color: ${({ theme }) => theme.subText};
+  position: absolute;
+  left: -8px;
+  top: -8px;
+  opacity: 0;
+  transition: all 0.1s linear;
+  z-index: 13;
+  cursor: pointer;
+  :hover {
+    background-color: ${({ theme }) => theme.tableHeader};
+  }
+`
+
+const WidgetWrapper = styled.div<{ show?: boolean }>`
+  position: fixed;
+  right: 0;
+  top: 110px;
+  transition: all 1.5s ease-out;
+  z-index: 20;
+  ${({ show }) =>
+    !show &&
+    css`
+      right: -40px;
+      visibility: hidden;
+    `}
+  :hover {
+    ${CloseButton} {
+      opacity: 1;
+    }
+  }
+
+  ${({ theme }) => theme.mediaWidth.upToSmall`
+    
+  `}
+`
+const ButtonWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  background-color: ${({ theme }) => theme.tableHeader};
+  border-radius: 8px 0 0 8px;
+  overflow: hidden;
+  box-shadow: 0 0 4px 2px #00000040;
+`
+const IconButton = styled.div`
+  padding: 12px;
+  background-color: ${({ theme }) => theme.tableHeader};
+  cursor: pointer;
+  transition: all 0.2s ease;
+  > * {
+    height: 16px;
+    width: 16px;
+  }
+  :hover {
+    filter: brightness(0.8);
+  }
+`
+const ExpandedWidgetWrapper = styled.div<{ show?: boolean }>`
+  position: fixed;
+  right: 0;
+  top: 110px;
+  background-color: ${({ theme }) => theme.tableHeader};
+  border-radius: 20px 0 0 20px;
+  z-index: 1000;
+  transition: all 0.2s ease;
+  transform: ${({ show }) => (show ? css`translateX(0)` : css`translateX(100%); visibility:hidden;`)};
+  overflow: hidden;
+  min-height: 450px;
+`
+
+const Tab = styled.div<{ active?: boolean }>`
+  width: 25%;
+  min-width: 168px;
+  padding: 12px 20px;
+  font-size: 14px;
+  line-height: 24px;
+  white-space: nowrap;
+  display: flex;
+  justify-content: center;
+  cursor: pointer;
+  border-bottom: 2px solid transparent;
+  transition: all 0.2s ease;
+  ${({ active, theme }) =>
+    active
+      ? css`
+          color: ${theme.text};
+          background-color: ${rgba(theme.primary, 0.3)};
+          border-bottom: 2px solid ${theme.primary};
+        `
+      : css`
+          background-color: ${({ theme }) => theme.tableHeader};
+        `}
+
+  > * {
+    text-align: center;
+    width: fit-content;
+  }
+  :hover {
+    filter: brightness(1.2);
+  }
+`
+
+const TextButton = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 8px;
+  font-size: 14px;
+  line-height: 20px;
+  font-weight: 500;
+  cursor: pointer;
+  :hover {
+    filter: brightness(1.2);
+  }
+`
+
+export enum WidgetTab {
+  MyWatchlist = 'My Watchlist',
+  Bullish = 'Bullish',
+  Bearish = 'Bearish',
+  TrendingSoon = 'Trending Soon',
```

Comment:
```suggestion
  MyWatchlist = t`My Watchlist`,
  Bullish = t`Bullish`,
  Bearish = t`Bearish`,
  TrendingSoon = t`Trending Soont`,
```

Reply:
Actually we should not translate like this, we should have a map object with the keys is this enum and value are the translated text

Reply:
will find a better way later